framework/build.xml

```diff
@@ -64,7 +64,7 @@
 
     <target name="compile" description="compile without cleaning">
         <mkdir dir="classes"/>
-        <javac encoding="utf-8" srcdir="src" destdir="classes" debug="true" source="1.8" target="1.8">
+        <javac encoding="utf-8" srcdir="src" destdir="classes" debug="true">
```

Comment:
Why was it needed? I guess it's still a good idea to explicitly declare that Play itself is runnable with Java8 (even if somebody occasionally builds it with Play9+).

Reply:
@asolntsev thank you for your comments !
I wanted to give more freedom to choose the source and target versions by specifying them as environment variables, especially to test compatibility with building with jdk9 or 10.
Indeed it may be a good idea to keep target and sources at "1.8" for now, I'll make these changes.